inst/templates/gitlab-ci.yml

```diff
@@ -0,0 +1,46 @@
+image: rocker/tidyverse
+
+stages:
+  - build
+  - test
+  - deploy
+
+building:
+  stage: build
+  script:
+    - R -e "remotes::install_deps(dependencies = T)"
+    - R -e 'devtools::check()'
+
+# To have the converage percentge appear as a gitlab badge follow these instructions
```

Comment:
As mentioned below I think a better solution is to just use `covr::gitlab(quiet = FALSE)`. In that case you would modify your coverage string to be `Coverage: \d+\.\d+`

Reply:
Thanks for the advice on that, I think when I originally wrote this, I had a reason for doing it that way, but in the end, it doesn't make much sense. I'll move the resolution to https://github.com/r-lib/usethis/issues/738